File: tests/fixtures/python-project/app/auth/jwt.py
import jwt
from app.utils.config import get_config

SECRET = "dev-secret"

def verify_token(token: str) -> dict:
    config = get_config()
    return jwt.decode(token, config["jwt_secret"], algorithms=["HS256"])

def sign_token(payload: dict) -> str:
    config = get_config()
    return jwt.encode(payload, config["jwt_secret"], algorithm="HS256")

File: tests/fixtures/sample-project/src/auth/jwt.ts
import jwt from 'jsonwebtoken';
import { getConfig } from '../utils/config.js';

export interface JWTPayload {
  userId: string;
  role: string;
}

export function verifyToken(token: string): JWTPayload {
  const config = getConfig();
  return jwt.verify(token, config.jwtSecret) as JWTPayload;
}

export function signToken(payload: JWTPayload): string {
  const config = getConfig();
  return jwt.sign(payload, config.jwtSecret, { expiresIn: '24h' });
}
